### ISSUE TITLE
[BUG] Ubuntu 22.04.1 LTS lots of notifications "Bing Wallpaper of the Day for none"

### ISSUE THREAD

**Issue by digeomel** [neffo/bing-wallpaper-gnome-extension#195]
**Describe the bug**
I switched recently from Ubuntu 18.0.4 to 22.04.1 LTS and the extension behaves erratically. For one, I'm not sure it is picking up the correct background for the day, seems to be totally random, and when it seems to fetch the latest I get lots of notifications with the message "Bing Wallpaper of the Day for none" and different photos.

**Screenshots**
![image](https://user-images.githubusercontent.com/3295889/215045453-99ae46c5-20f5-4819-baf2-2f5121ad733c.png)

![image](https://user-images.githubusercontent.com/3295889/215045767-7c7d8a9b-9452-4789-87f9-076c6881773d.png)

**To Reproduce**
There are no steps to reproduce, this seems to happen when the extension tries to refresh the desktop background.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04.1 LTS
 - Extension Version: 44
 - GNOME Version: 42.5

"Always show new images" is set to ON.
"Enable all features on Wayland" is set to ON.
"Selected image" is "Most recent image".
"Bing locale" was "auto", switched it to "English (United Kingdom)", no difference. I am in Belgium.

**Comment by neffo** [comment]
Possible change to the shape of the JSON data, I'll look into it.

**Comment by neffo** [comment]
Yeah, looks like this bug was introduced recently in the extension (not source JSON). Thanks for the bug report, will correct in next version (45). In the meantime disabling notifications should be enough.

Are you seeing multiple notifications after having your PC off for a few days? That is expected behavior (though maybe not what people want). It notifies for any NEW image, not just that day, so you might get 7 additional notifications.

**Comment by digeomel** [comment]
Hi, thanks for the quick reply! 👍🏼 
I see multiple notifications every time it tries to fetch the latest. I turn off this laptop every day in the evening and turn it back on the next morning.
